src/core/reactor.ts

```diff
@@ -956,15 +956,15 @@ protected _getFirstReactionOrMutation(): Reaction<any> | undefined {
             // Rule out write conflicts.
             //   - (between reactors)
             if (this._dependencyGraph.getBackEdges(dst).size > 0) {
-                return false;
+                throw Error("Destination port is already occupied.")
             }
 
-            //   - between reactors and reactions (NOTE: check also needs to happen
-            //     in addReaction)
-            var deps = this._dependencyGraph.getEdges(dst) // FIXME this will change with multiplex ports
-            if (deps != undefined && deps.size > 0) {
-                return false;
-            }
+            // //   - between reactors and reactions (NOTE: check also needs to happen
+            // //     in addReaction)
+            // var deps = this._dependencyGraph.getEdges(dst) // FIXME this will change with multiplex ports
+            // if (deps != undefined && deps.size > 0) {
+            //     return false;
+            // }
```

Comment:
Same here please clean up the commented lines.

Reply:
It was reflected in https://github.com/lf-lang/reactor-ts/pull/108/commits/9019da3204bf0e1773451d2c1a27b0485e3ac8e1